bundles/framework/divmanazer/component/VisualizationForm.js

```diff
@@ -202,6 +227,8 @@ Oskari.clazz.define(
             var formClazzes = this._getFormClazz();
             var fClazzName;
             var fClazz;
+            this._options = jQuery.extend(this._options, { name: featureStyle.name });
```

Comment:
The name is part of the style definition here too.

Reply:
Name is now passed as a new variable to function.